docs/apps/matlab.md

```diff
@@ -176,6 +176,12 @@ We can configure MPS on a local computer using the following instructions.
 
 
 #### Linux and MacOS
+Step 2: Run in shell:
```

Comment:
Why do these start at "Step 2:", not "Step 1:"?

Reply:
Added.